src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java

```diff
@@ -27,7 +27,7 @@
 
 /**
  * <p>
- * Check nested (inner) classes/interfaces are declared at the bottom of the
+ * Checks nested (internal) classes/interfaces are declared at the bottom of the
  * class after all method and field declarations.
```

Comment:
> bottom of the class

Should this be something like bottom of the primary, or maybe top-level class?

Reply:
done